bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateSystemSelection.js

```diff
@@ -0,0 +1,68 @@
+Oskari.clazz.define('Oskari.coordinatetransformation.component.CoordinateSystemSelection',
+    function (view) {
+        this.instance = view;
+        this.loc = view.loc;
+        this.element = null;
+        this.select = Oskari.clazz.create('Oskari.coordinatetransformation.component.select', view );
+        this.selectInstance = null;
+        this.dropdowns = null;
+        this._template = {
+            systemWrapper: jQuery('<div class="coordinateSystemWrapper"></div>'),
+            coordinatSystemSelection: _.template(
+                '<div class="transformation-system">' +
+                    '<h5><%= title %></h5>'+
+                    '<div class="geodetic-datum"><b class="dropdown_title"><%= geodetic_datum %></b> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div> </div> </br> ' +
+                    '<div class="coordinate-system"><b class="dropdown_title"><%= coordinate_system %></b> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div>  </div> </br> ' +
+                    '<div class="map-projection" style="display:none;"> <%= map_projection %> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div> </div> </br>' +
+                    '<div class="geodetic-coordinatesystem"><b class="dropdown_title"><%= geodetic_coordinate_system %> *</b> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div> </div> </br> ' +
+                    '<div class="height-system"><b class="dropdown_title"><%= height_system %></b></div> <a href="#"><div class="infolink icon-info"></div></a> <div class="select"></div> '+
+                '</div>'
+            ),
+        }
+        if (this.element === null) {
```

Comment:
Isn't element always null here (assigned a couple of lines above)? Why do we need to check it?